incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -339,17 +346,25 @@ As of August 2020, the most recent official release is v0.5.1.
 # Set the desired release:
 HNC_VERSION=v0.5.1
 
+# Set the desired OS and architecture (from v0.5.2)
+HNC_PLATFORM=linux_amd64
+
 kubectl apply -f https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/hnc-manager.yaml
 ```
 
-#### Download the kubectl plugin (Linux only)
+#### Download the kubectl plugin
 
 The `kubectl-hns` plugin makes most HNC use and administration much easier; we
 strongly recommend installing it. Run the following commands from any directory
 on your `PATH`.
 
 ```bash
+# Download the plugin
 curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hns -o ./kubectl-hns
+
+# Download the plugin (from v0.5.2)
+curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hns_$HNC_PLATFORM} -o ./kubectl-hns
+
 chmod +x ./kubectl-hns
 ```
```

Comment:
This section is now sufficiently complicated that I don't think it makes sense to repeat it twice. Instead of a full separate section, can you just add a link to `#use-prepare` to reference the earlier section?

Reply:
Ok! Make sense. Let me change it.